### ISSUE TITLE
Moore foundation grant final report

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/team-compass#183]
Hey all - as many of you know, a few years back the Binder Project got its first (and only...) grant to support development and maintenance of BinderHub, repo2docker, and mybinder.org.

This was a 1-year grant, and it wound down in May. To close this loop, we need to write a short report for the Moore Foundation that has three sections: **a list of papers, awards, impact, etc; a narrative document describing what we did with the grant; an updated budget saying what we actually spend out money on.**

I've put links to each of these documents in a:

### **[master to-do document here](https://docs.google.com/document/d/1oa0YFGtDxPBicLZo8l_tOOa3cHvCz-ogm6rNtMeYHpE/edit#)**

Over the next week or two, I will spend time filling this out. **It would be fantastic if others on the team, particularly those who benefited from this grant, could help out**. In particular @betatim, @minrk,  @willingc, and @yuvipanda (gosh it feels like ages ago!)

I think that if we work together to fill in pieces of information that we know, this shouldn't be too much work! Feel free to fill in whatever information you think is useful/relevant, and I am happy to weave the whole thing into a narrative that is consistent etc.

**Comment by minrk** [comment]
I added some templates, references, and links to the narrative, and entered the actual Simula funds in the budget and data collection.

**Comment by choldgraf** [comment]
Hey all - I took a stab [at the narrative document](https://docs.google.com/document/d/18GaSNRfdLmwAYtjiDzlSCaDGhrUXkK5r24XGNhQR_vc/edit#) and filled in as much information as I could. It'd be helpful if somebody could take a look at that to let me know how it feels. There are several sections that don't necessarily have a single right answer, and I'd appreciate if you all would give your comments, edits, thoughts, etc.

For the "data" side of things, we need information from @willingc and @betatim I believe (I tried to fill in your information as best I could but let me know if I've missed something). In addition, @fperez should indicate whether there is anybody paid off of this grant that we haven't mentioned here.

**Comment by willingc** [comment]
@choldgraf Looks good. I've filled all that I can on the data/personnel/conferences/outputs. Ana/Brian or someone at Cal Poly will have to get the budget numbers for my work. Thanks for doing this Chris.

**Comment by choldgraf** [comment]
thanks Carol! @Ruv7 or @ellisonbg any chance you can look over the budget numbers (see the links above) to confirm they are correct?

**Comment by betatim** [comment]
@choldgraf with "data side of things" do you mean https://docs.google.com/spreadsheets/d/1TxQOOo2WSv5KWGO_13oEn0tNAVWegcTt/edit#gid=1122195543 ? I didn't receive any prizes, didn't publish beyond what we did for the project and none of the conference talks I gave were funded by Moore. I added the Mozilla mini grant I got though.

**Comment by choldgraf** [comment]
@betatim yep, I just meant the "entering things into spreadsheets" part of it :-)

thanks for your comments on the document!

also, can I just say that looking at the document we put together makes me realize how much we have all accomplished with this project, especially given the relatively small number of resources that we have at our disposal. Who would have thought you can grow a public service that serves nearly 300,000 people a month with just a single 1.5 year grant? <3

**Comment by choldgraf** [comment]
Another thought - once we've got the report finished up, how do people feel about posting this online at a Zenodo DOI? We can create a repository for it and store the grant (with $$$ and other sensitive information scrubbed) there so that others can take a look.

(if anybody knows a better way to get a zenodo DOI other than creating a new repo, I'm all ears)

**Comment by betatim** [comment]
You can directly upload files to Zenodo via their UI and get a DOI. I am fine with putting it on Zenodo/git repo.

**Comment by willingc** [comment]
Fine by me. Thanks @choldgraf.

**Comment by choldgraf** [comment]
hey all - FYI we've submitted this! yay 🎉 once Moore gets back and says it looks good, I'll post it to a Zenodo repo

**Comment by choldgraf** [comment]
Hey all - today I put together two documents in our Moore proposal and turned them into a Zenodo DOI. The link below contains a PDF of our original proposal to Moore, as well as the "end of grant" narrative document (nothing about $$$, just the "what we proposed" and "what we did" pieces.

https://zenodo.org/record/3372810

Take a look and let me know if I missed something! I chose authors based on the current jupyterhub/binder teams on the team compass, as well as those who weren't on those teams but are listed on the original grant proposal.

If folks are happy, we can tell people about it and then close this issue! 🎉
